tests/helpers.py

```diff
@@ -2,22 +2,28 @@
 from pathlib import Path
 from pprint import pprint
 from typing import Optional, List
-from frictionless import validate_package
+from frictionless import validate
+
 
 THIS_SCRIPT_PATH = Path(__file__).parent
 REPOSITORY_ROOT_PATH = THIS_SCRIPT_PATH / ".."
 EXAMPLE_PATH = REPOSITORY_ROOT_PATH / "example" / "datapackage.json"
 PROFILE_PATH = REPOSITORY_ROOT_PATH / "camtrap-dp-profile.json"
-RELAXED_PROFILE_PATH = REPOSITORY_ROOT_PATH / "camtrap-dp-profile-relaxed.json" # File created in later step
```

Comment:
I would leave this as is (single line) to keep it consistent with the variables above.

Reply:
I understand this is done automatically by "black" formatting because the line is too long.